### PR TITLE
Pokemon typos

### PR DESCRIPTION
**Pull request by OverlordQ**

### CODE REVIEW

**Comment by miracle091** [comment]
Ok this is working :smile:

File: PokemonGo.RocketAPI.Console/Configs/ConfigPokemonsToKeep.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Dragonite
+Venusaur
 Charizard
 Blastoise
 Alakhazam
```
Comment (miracle091) [inline PokemonGo.RocketAPI.Console/Configs/ConfigPokemonsToKeep.txt]: This should be: Alakazam

Comment (OverlordQ) [inline PokemonGo.RocketAPI.Console/Configs/ConfigPokemonsToKeep.txt]: Then the proto needs updated:

```
ALAKHAZAM = 65;
```

**Comment by miracle091** [comment]
See this #193

**Comment by NecronomiconCoding** [comment]
apprroved #193 , feel free to update yours

**Comment by NecronomiconCoding** [comment]
would be nice if you could fix the conflicts.

**Pull request closed without merging**